{"uuid": "612da7b2-0742-4357-b4f0-e2c83d36f579", "children": ["c11bb9ce-b345-4c8e-8ca9-28484367d983", "6eb46168-bd2a-4d86-930b-cc0148032d4a", "0ea2cdb6-0b83-4e51-a658-4c1bf57fd870", "9827e0f0-b35d-44d0-abed-17ca6051076d", "2bec246b-29c3-4446-9ef8-acca0977451d", "1ed08d63-2c38-442d-8afb-4e0f702a1b24", "8595d72b-cc85-44ef-800e-595fdb6c4d35", "775560b1-efd3-46c8-bcd8-8fab59e9da3d", "2420a095-1dbf-4e6f-9656-e64a5dff13cf"], "befores": [{"name": "get_job_details", "status": "broken", "statusDetails": {"message": "requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pluggy-1.5.0-py3.10.egg/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pytest-7.4.3-py3.10.egg/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pytest-7.4.3-py3.10.egg/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/Test_API_Suite/test_corteva.py\", line 16, in get_job_details\n    peviitor_jobs_data = testutils.scrape_peviitor(company_name, 'Rom√¢nia')\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/utils.py\", line 70, in scrape_peviitor\n    response_data = TestUtils._get_request(params)\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/utils.py\", line 48, in _get_request\n    updateapi.get_token()\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/sites/__utils/peviitor_update.py\", line 42, in get_token\n    self.access_token = requests.request(\"POST\", \"https://api.peviitor.ro/v5/get_token/\", headers=post_header, data=payload).json()['access']\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/requests-2.28.2-py3.10.egg/requests/models.py\", line 975, in json\n    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)\n"}, "start": 1714064040801, "stop": 1714064044895}], "start": 1714064040801, "stop": 1714064045255}